edx_solutions_organizations/models.py

```diff
@@ -86,6 +86,19 @@ def get_all_users_by_organization_attribute_filter(users, organizations, attribu
         return users
 
 
+class OrganizationUserMapping(TimeStampedModel):
+    """
+    The OrganizationUserMapping model contains information describing the
+    link between a particular user, organization and it's main company flag.
+    """
+    user = models.ForeignKey(User, related_name="user_organizations", on_delete=models.CASCADE)
+    organization = models.ForeignKey(Organization, related_name="user_organizations", on_delete=models.CASCADE)
+    is_main_company = models.BooleanField(default=False)
```

Comment:
did you considered adding `is_main_company` column in `OrganizationUsersAttributes` as an attribute? 
or in `user_organization` table because this table as similar structure.?